src/page-building/career/culture.tsx

```diff
@@ -24,69 +19,38 @@ const TeaserGrid = styled.div`
   }
 `;
 
-const ASPECTS: CultureAspect[] = [
-  {
-    title: 'Cross-Functional',
-    description:
-      'Teams aus Designern, Produktmanagern, Entwicklern, Supportmitarbeitern, Marketingspezialisten usw. bringen die richtige Mischung aus Know-how und Erfahrung mit und sorgen dafür, dass auch mal über den Tellerrand geschaut wird.',
-    illustration: 'radar_030',
-  },
-
-  {
-    title: 'Self-Organisation',
-    description:
-      'Jedes Team bei Satellytes ist für seine eigenen Aufgaben verantwortlich. In den Worten des Agilen Manifests: „Die besten Architekturen, Anforderungen und Entwürfe entstehen durch selbstorganisierte Teams.“',
-    illustration: 'rocket_011',
-  },
-
-  {
-    title: 'Agile',
-    description:
-      'Schnelles Feedback, kurze Entscheidungswege und Offenheit für Veränderung sind uns wichtig – egal, ob man es Scrum, Kanban oder XP nennt.',
-    illustration: 'sputnik_045',
-  },
-
-  {
-    title: 'Erfolgreich scheitern',
-    description:
-      'Wenn wir etwas Neues ausprobieren, riskieren wir immer auch einen Misserfolg. Macht nichts – aus Fehlern lernen, Erkenntnisse mit anderen teilen und im nächsten Projekt in Erfolge ummünzen.',
-    illustration: 'planets_005',
-  },
-
-  {
-    title: 'Feedback',
-    description:
-      'Zusammenarbeit ist bei uns nicht nur ein Wort: Code-Review, Inline-Kommentare, Pair Programming, tägliche Stand-up-Meetings, 360-Grad-Feedback usw.',
-    illustration: 'report_031',
-  },
-  {
-    title: 'Open Work',
-    description:
-      'Offen sein, zuhören, andere Meinungen respektieren und gemeinsam die besten Lösungen finden.',
-    illustration: 'book_038',
-  },
```

Comment:
oh that's not good. you should keep it explicit and repeat yourself. the current code is very implicit and worse the key part of the iteration are the remaining  icons. That's confusing and I would prefer to see the translations here.

This is not possible out of the box but you can wrap the list with a function to pass in `t`.

```
const ASPECTS: CultureAspect[] = t => [
 {
    title: 'Cross-Functional',
    description: t("something"),
    illustration: 'radar_030',
  },

]


// later use 
ASPECTS(t)
```


that's pretty much a workaround but it helps to preserver the readability. Can you do that? Thank you!